notebooks/05_choropleth.md

```diff
@@ -513,150 +528,125 @@ results by comparing the number of values assigned to each class across the
 
 
-### Sequential Color Schemes
+### Sequential Palettes
 
 
-Our attribute is measured in dollars and is characterized as a *sequential* attribute. To choose an appropriate sequential scheme we can override the `cmap` parameter:
+Sequential color schemes are appropriate for continuous data where the origin is in one end of the series. The `PCGDP1940` column we have been using so far is a good example. In these cases, we want a palette that encodes this features in its choice of colors. Sequential palettes use a gradient of colors from an origin color to a destination color. The example above, where lowest values are encoded in the lightest yellow and the highest in dark green is a good one. Sequential palettes can also have a shades of a single color. For example, the popular "blues" palette in Color Brewer is a great choice too:
 
-```python caption="Sequential Color Map (Blues). Quantile Classification, Mexcian PCGDP1940." tags=[]
-f, ax = plt.subplots(1, figsize=(9, 9))
-mx.plot(ax=ax, column='PCGDP1940', legend=True, scheme='Quantiles', legend_kwds={'fmt':'{:.0f}'}, \
-         cmap='Blues')
-ax.set_axis_off()
-ax.set_title('PCGDP1940')
-plt.axis('equal')
-plt.show()
+```python caption="Quantile choropleth with black borderlines, Mexican State PCGDP1940." tags=[]
+ax = mx.plot(
+    column='PCGDP1940',          # Data to plot
+    scheme='Quantiles',          # Classification scheme
+    cmap='Blues',                # Color palette
+    edgecolor='k',               # Borderline color
+    linewidth=0.1,               # Borderline width
+    legend=True,                 # Add legend
+    legend_kwds={'fmt':'{:.0f}'} # Remove decimals in legend (for legibility)
+)
+ax.set_axis_off();
 ```
 
-which now uses a single-hue sequential color map with the lighter shades representing lower values. One difficulty with this map is that the poor states in the southern portion of Mexico blend into the background of the map display. This can be adjusted by overriding the `edgecolor`:
-
+Note how, in this case, we switch borderlines to black so that we can distinguish states in the lowest category from the white background.
 
-```python caption="Modification of  polygon edge color for clarity. Quantile Classification, Mexcian PCGDP1940." tags=[]
-f, ax = plt.subplots(1, figsize=(9, 9))
-mx.plot(ax=ax, column='PCGDP1940', legend=True, scheme='Quantiles', legend_kwds={'fmt':'{:.0f}'}, \
-         cmap='Blues', edgecolor='k')
-ax.set_axis_off()
-ax.set_title('PCGDP1940')
-plt.axis('equal')
-plt.show()
-```
 
-### Diverging Color Schemes
+### Diverging Palettes
 
-A slightly different type of attribute is the so-called "diverging" values attribute. This is
-useful when one wishes to place equal emphasis on mid-range critical values as
+A slightly different type of palette is the so-called "diverging" values attribute. This is
```

Comment:
s/attribute\./pallete\./

Or merge the first two sentences:

A slightly different type of palette is the so-called "diverging" pallet which is useful with...